v1.1/deploy-cockroachdb-on-aws-insecure.md

```diff
@@ -252,9 +252,11 @@ To test this, install CockroachDB locally and use the [built-in SQL client](use-
 
 4. Check which node you were redirected to:
```

Comment:
Totally agree that we need to use includes throughout these tutorials. I'd prefer to handle that in a separate PR, though. There are many minor differences between cloud deployment guides that need to be considered when breaking them up into includes.